SPECS/rubygem-connection_pool/rubygem-connection_pool.spec

```diff
@@ -1,16 +1,17 @@
+# Disabled debuginfo package as the autogenerated 'debugfiles.list' file is empty.
```

Comment:
Can we please link a test pipeline build to this as well :)

Reply:
I didn't run any test build pipelines but you're right. I'll run and link one soon.